src/DynamoPackages/PackageLoader.cs

```diff
@@ -76,9 +76,20 @@ public string DefaultPackagesDirectory
 
         private readonly List<string> packagesDirectoriesToVerifyCertificates = new List<string>();
 
+        // The standard library directory is located in the same directory as the DynamoPackages.dll
+        private string StandardLibraryDirectory =>
+            Path.Combine(Path.GetDirectoryName(Assembly.GetAssembly(GetType()).Location),
+                @"Standard Library", @"Packages");
+
         public PackageLoader(string overridePackageDirectory)
-            : this(new[] { overridePackageDirectory })
         {
+            if (overridePackageDirectory == null)
+                throw new ArgumentNullException("overridePackageDirectory");
+
+            packagesDirectories.Add(overridePackageDirectory);
```

Comment:
add `this` to be consistent. Or remove it everywhere..

Reply:
What is our preference, use `this` or not using `this`?

Reply:
Not using.

Reply:
Thanks